enterprise/course_catalog_api.py

```diff
@@ -64,6 +64,30 @@ def get_all_catalogs(self):
         """
         return self._load_data('catalogs', default=[])
 
+    def get_single_catalog_courses(self, catalog_id):
+        """
+        Return the courses included in a single course catalog by ID.
+
+        Args:
+            catalog_id (int): The catalog ID we want to retrieve.
+
+        Returns:
+            dict: Courses of the catalog in question
+        """
+        return self._load_data('catalogs/{catalog_id}/courses'.format(catalog_id=catalog_id))
+
+    def get_single_course_details(self, course_key):
```

Comment:
Here as well -- why `get_single_course_details` instead of `get_course_datails'?